src/libcode/vx_tc_util/track_info.cc

```diff
@@ -1138,6 +1162,52 @@ TrackInfo consensus(const TrackInfoArray &tracks,
       // Compute consensus CycloneLevel
       if(!is_bad_data(pavg.v_max())) pavg.set_level(wind_speed_to_cyclonelevel(pavg.v_max()));
 
+      // Loop over the diag name and the input track points and get consensus diag value
+      // (mean value across all members that have the diag value)
+      for (j=0; j<tavg.diag_name().n(); j++){
+         mlog << Debug(3) << "TrackInfo::consensus(). Working on calculating mean values for tavg.diag_name()[" << j << "] = " << tavg.diag_name()[j] << "\n";
```

Comment:
```suggestion
         mlog << Debug(4) << "Computing consensus \"" << tavg.diag_name[j] << "\" 
 diagnostic value.\n";
```
Making log message more concise and increasing to verbosity level 4.